content/en/blog/docker-multi-arch/index.md

```diff
@@ -0,0 +1,114 @@
+---
+date: 2021-09-20
+title: "The Arm Adventure on Docker"
+linkTitle: "The Arm Adventure on Docker"
+description: "Since the Findy Agency project launched, Docker has been one of our main tools to help set up the agency development and deployment environments. Unexpected headache developed when our colleague purchased a M1 Mac and our images refused to run on ARM platform."
+author: Laura Vuorenoja
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+Since the Findy Agency project launched, Docker has been one of our main tools to help set up the agency development and deployment environments. First of all, we use Docker images for our cloud deployment. On a new release, the CI build pipeline bundles all needed binaries to each service image. After the build, the pipeline pushes Docker images to [the GitHub container registry](https://github.blog/2020-09-01-introducing-github-container-registry/), from where the deployment pipeline fetches them and updates the cloud environment. 
+
+{{< imgproc agency-deployment-pipeline Fit "925x925" >}}
+<em>Agency deployment pipeline: New release triggers image build in GitHub actions. When the new image is in the registry, AWS Code Pipelines handles the deployment environment update.</em>
+{{< /imgproc >}}
+
+
+In addition, we've used Docker to take care of the service orchestration in a local development environment. Agency consists of three different services, and if one wishes to set agency fully up on a local desktop, it is straightforward by executing a docker-compose script. The script pulls correct images to the local desktop and sets needed configuration parameters. Without the orchestration, setting up and updating the three services would be cumbersome.
```

Comment:
.. cumbersome? (not in all of the cases)

related to my previous comment and especially the development environment: I personally think that it's very important to offer an excellent debugging environment. And in our case, we can still debug core-agency together with the auth server pretty easily. There are own scripts for that for both of them. It also means that we don't have to set the whole environment up in all of the cases.

My point: we should give the message that even the docker is important, it's not mandatory, or even needed in all of the dev cases?